tools/PGODatabase/NuSpecs/build/PGO.targets

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+     <ProfileGuidedDatabaseFileName>$(MSBuildThisFileDirectory)..\tools\$(PlatformShortName)\$(ProjectName).pgd</ProfileGuidedDatabaseFileName>

```

Comment:
WHOA. This depends on a variable having been set outside the package (!)

Can the package itself derive PlatformShortName?

Reply:
sure that's a good idea. copy pasta'd the code.